src/test/java/codeu/controller/UserVerificationTest.java

```diff
@@ -0,0 +1,35 @@
+package codeu.controller;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class UserVerificationTest {
+    private UserVerificationServlet UserVerificationServlet;
+    private HttpServletRequest mockRequest;
+    private HttpServletResponse mockResponse;
+    private RequestDispatcher mockRequestDispatcher;
+
+    @Before
+    public void setup() throws IOException {
+        UserVerificationServlet = new UserVerificationServlet();
+        mockRequest = Mockito.mock(HttpServletRequest.class);
+        mockResponse = Mockito.mock(HttpServletResponse.class);
+        mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
+        Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/userVerification.jsp"))
+                .thenReturn(mockRequestDispatcher);
+    }
+    @Test
+    public void testDoGet() throws IOException, ServletException {
+        UserVerificationServlet.doGet(mockRequest, mockResponse);
+
+        Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
+    }
+}
```

Comment:
hmm maybe make the instance attribute first letter not capitalized? like userVerificationServlet instead of UserVerificationServlet so Java doesn't confuse the instance attribute with the class name

Reply:
yeah I realized that right after I asked for the push lol I changed it and i'll send another push